### DISCUSSION TITLE
How do I play

### DISCUSSION THREAD

**Discussion by Poc2000** [tatapuchi/Idle-Developer#69] [Q&A] [answered]
I like just now got into github and I have only been playing games where they give me a link so I was wondering how I can play and also is it available on browser or just through a certain software? I feel kind of dumb asking this but I am completing lost

**Comment by tatapuchi** [comment, marked as answer]
You just neecd visuual stduio with xamarin dev tools, then you can clone this repo and try the game out.

**Reply by tatapuchi** [reply]
I can help out if u need.

**Reply by Poc2000** [reply]
Ok so then I need an actual computer, I am stuck using a Chromebook that I got a while ago since my computer is broken right now and I am getting it fixed. And i would love your help once It gets fixed so I can let you know when that happens.

**Reply by tatapuchi** [reply]
Yep, I'm sorry, hope your computer gets fixed soon

**Reply by Poc2000** [reply]
no problem and thank you
